src/components/bootstrap/bootstrap.css

```diff
@@ -0,0 +1 @@
+@import url('https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css');
```

Comment:
as we are including this as a dependency,  we should be able to pull this right out of _node_modules_ like so
```
@import url('~/bootstrap/path/to/dist.css')
```
The `~` will be recognized by **css-loader** and resolve to _node_modules_